eth_abi/utils/parsing.py

```diff
@@ -37,6 +37,17 @@ def process_strict_type(typ):
             raise ValueError("Integer size out of bounds")
         if int(sub) % 8 != 0:
             raise ValueError("Integer size must be multiple of 8")
+    # Check validity of fixed type
+    elif base == 'ufixed' or base == 'fixed':
+        if not re.match('^[0-9]+x[0-9]+$', sub):
+            raise ValueError("Fixed type must have suffix of form <high>x<low>, eg. 128x128")
+        bits, minus_e = [int(x) for x in sub.split('x')]
```

Comment:
just a matter of preference when splitting something into exactly two pieces.

```python
bits, _, minus_e = sub.partition('x')
```

Reply:
oh, grumble, the casting to integer makes that more complex.  Ignore my suggestion.  Good to merge as-is.